ras_backstage/controllers/uaa_controller.py

```diff
@@ -63,17 +38,10 @@ def sign_in(username, password):
         raise ApiError(url, response.status_code)
 
     try:
-        decoded_jwt = jwt.decode(
-            response.json()["access_token"],
-            algorithms=response.json().get('alg'),
-            verify=True,
-            key=get_public_key(),
-            audience='ras_backstage',
-            leeway=10,
-        )
-
         logger.debug('Successfully retrieved UAA token')
-        return decoded_jwt
+        token = response.json()
```

Comment:
Same as above, if the json is malformed or missing then you get a valueError, should be this be handled too?

Reply:
I'll add an except block